server/apps/content_types/content_types.py

```diff
@@ -0,0 +1,30 @@
+
+import superdesk
+
+
+CONTENT_TYPE_PRIVILEGE = 'content_type'
+
+
+class ContentTypesResource(superdesk.Resource):
```

Comment:
Can we add a brief summary of what ContentType resource is all about?
(one or two sentences in the class docstring should already be enough)